wandb/sdk/launch/_project_spec.py

```diff
@@ -455,7 +457,9 @@ def get_entry_point_command(
     return entry_point.compute_command(parameters)
 
 
-def create_project_from_spec(launch_spec: Dict[str, Any], api: Api) -> LaunchProject:
+def create_project_from_spec(
+    launch_spec: Dict[str, Any], api: Api, rqi_id: str = None
+) -> LaunchProject:
```

Comment:
this is a gross change, totally breaks the idea of `create_project_from_spec`, but we probably don't want to add the rqi_id to the launch_spec as it would be visible to the user :/

Reply:
I guess the alternative here is to update the launch project with the run queue item id in the agent loop. I think that's slightly better since it is more explicit that the launched run will only have a queue when working with a launch agent.